test/server/paymentProviders/stripe/zero-decimal-currencies.test.js

```diff
@@ -0,0 +1,72 @@
+import { expect } from 'chai';
+import gql from 'fake-tag';
+import { describe, it } from 'mocha';
+
+import * as store from '../../../stores';
+import * as utils from '../../../utils';
+
+describe('server/graphql/v1/transaction', () => {
```

Comment:
We usually put GraphQL tests in the `test/server/graphql/v1` folder

We want to make the `describe` match the filename.

Reply:
Move to the `v1 folder and change the names. Sorry I missed this somehow. 😄